framework/test/unit/modules/fee/module.spec.ts

```diff
@@ -187,6 +220,9 @@ describe('FeeModule', () => {
 		});
 
 		it('should burn the used fee when addressFeePool is not defined', async () => {
```

Comment:
I think this should be broken into two cases:
1) If `addressFeePool` is not defined and user account for token id does not exists for generator address: burn entire fee.
2) If `addressFeePool` is not defined and user account for token id does exist for generator address: burn the used fee and transfer the remaining fee to the generator address.

These are options from lines 1 and 2 of the following table, respectively. Maybe this is not the nicest way to see the whole set of options, but...

|   | `addressFeePool`                | `generatorAddress`     | USED FEE                     | REMAINING FEE                  |
|---|---------------------------------|------------------------|------------------------------|--------------------------------|
| 1 | not defined                     | account does not exist | burn                         | burn                           |
| 2 | not defined                     | account exists         | burn                         | transfer to `generatorAddress` |
| 3 | defined, account does not exist | account does not exist | burn                         | burn                           |
| 4 | defined, account does not exist | account exists         | burn                         | transfer to `generatorAddress` |
| 5 | defined, account exists         | account does not exist | transfer to `addressFeePool` | transfer to `addressFeePool`   |
| 6 | defined, account exists         | account exists         | transfer to `addressFeePool` | transfer to `generatorAddress` |

Reply:
Many thanks for the feedback and descriptive table with all cases! I add a new test `should burn the entire fee when addressFeePool is not defined and user account of generator address does not exist for the token id` to cover option 1 from your comment.